lib/puppet_x/puppetlabs/cd4pe_client.rb

```diff
@@ -262,16 +262,29 @@ def web_ui_endpoint
       endpoint
     end
 
+    def connection_read_timeout
+      # Set timeout to 10min or customer override for our code deployment
+      # api as it is a long lived connection instead of polling for updates.
+      # If env var is specified and valid, override.
+      timeout = 600
+      env_var = ENV['CD4PE_MODULE_DEPLOY_READ_TIMEOUT']
```

Comment:
nit: this isn't the var, it's the value of the var 🙃

Reply:
also, if we have control over the name of this, which it looks like we do, consider adding "SEC" to the end of the name.

Reply:
based on zoom convo, decided not to change this in case anyone is utilizing this env var we don't want to change the value.

Reply:
but we are changing the name of the var